src/main/java/org/assertj/core/api/InstanceOfAssertFactories.java

```diff
@@ -89,7 +89,9 @@ public interface InstanceOfAssertFactories {
    *
    * @see #PREDICATE
    */
-  @SuppressWarnings({ "rawtypes", "unused" }) // rawtypes: using Class instance, unused: parameter needed for type inference
+  @SuppressWarnings({"rawtypes", "unused", "unchecked", "RedundantSuppression"})
+  // rawtypes+unchecked: using Class instance, unused: parameter needed for type inference.
+  // IntelliJ can warn that this is redundant when it is not.
```

Comment:
@ascopes do you happen to remember if this was happening when opening AssertJ code in IntelliJ, or when using these factories in client code?

At least, it seems that the former doesn't happen anymore.

Reply:
This was quite a long time ago. I can't remember seeing it more recently though so they probably have fixed it since!

Reply:
Great and thanks a lot for the quick feedback!

Reply:
No problem. Just so happened to pop up on my phone while I was doing something!